plugin/src/main/kotlin/me/gabber235/typewriter/entry/EntryDatabase.kt

```diff
@@ -44,9 +50,53 @@ object EntryDatabase {
 
 		this.entries = pages?.flatMap { it.entries } ?: listOf()
 
+		this.commandEvents = this.events.filter {
+			it::class.simpleName == "RunCommandEventEntry"
+		}
```

Comment:
This is very dirty. If you make a custom interface for it, like `CustomCommandEntry`. Then you can just check for `it is CustomCommandEntry`. Or even better, `this.events.filterIsInstance<CustomCommandEntry>()`